marketplace_builder/views/pages/sitemap.xml.liquid

```diff
@@ -0,0 +1,20 @@
+---
+slug: sitemap
+layout_name: ""
+static_cache:
+  expire: 3600
+metadata:
+  skip_sitemap:
+---
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
```

Comment:
I dont know what those google thingies are, but even google on their sitemap guide does not mention it.
```suggestion
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```

Sources:
https://www.sitemaps.org/protocol.html
https://support.google.com/webmasters/answer/183668?hl=en